testdata/templates/sql.tf

```diff
@@ -1,3 +1,8 @@
+provider "google" {
+  version     = "~> {{.Provider.version}}"
+  credentials = "{{.Provider.credentials}}"
```

Comment:
I think this prevents the usage of Application Default Credentials?

Could we make the injection of `credentials = ` conditional on an explicit `TEST_CREDENTIALS` file being provided?

Reply:
added.